components/Forms/Radio.tsx

```diff
@@ -0,0 +1,63 @@
+import { useRouter } from 'next/router'
+import { InputHTMLAttributes } from 'react'
+import { Tooltip } from '../Tooltip/tooltip'
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  keyforid: string
+  values: any[]
+  label: string
+  category: string
+}
+
+export const Radio: React.VFC<InputProps> = (props) => {
+  const { query } = useRouter()
+
+  return (
+    <>
+      <div className="radio mb-8" data-category={props.category}>
+        <span className="font-semibold inline-block mb-1.5">
+          <span className="text-danger">*</span>
+          <span className="mb-1.5 font-semibold text-content">
+            {' '}
+            {props.label}
+          </span>
+          <span className="text-danger font-bold ml-2">(required)</span>
+          <Tooltip field={props.name} />
+        </span>
+        {props.values.map((value, index) => (
+          <div key={index} className="flex items-center">
+            <input
+              type="radio"
+              id={`${props.keyforid}-${index}`}
+              name={`${props.keyforid}`}
+              value={correctForBooleans(value)}
+              className="hidden"
+              {...props}
+            />
+            <label
+              htmlFor={`${props.keyforid}-${index}`}
+              className="radio flex items-center"
+            >
+              <span className="w-6 h-6 inline-block mr-2 rounded-full border border-grey"></span>
+              <p>{value}</p>
+            </label>
+          </div>
+        ))}
+      </div>
+    </>
+  )
+}
+
+const correctForBooleans = (value: string) => {
+  switch (value) {
+    case 'Yes':
+      return 'true'
+      break
+    case 'No':
+      return 'false'
+      break
+    default:
+      return value
```

Comment:
if we reach this line, does that mean the boolean is invalid? and this should raise an error?

Reply:
No because this function is not called only for booleans. This question might also be rendering a regular radio question, in which you should just return the default value.

I guess technically to get rid of this check altogether we should never have boolean typed questions and they should be a radio with the values of 'Yes' and 'No'. 

Or maybe this is a checkbox question? Would require us to run it by Alla and Joan, what do you think?